gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/logback.xml

```diff
@@ -49,7 +49,7 @@
         <appender-ref ref="STDOUT" />
     </appender>
 
-    <logger name="io.gravitee" level="INFO" />
+    <logger name="io.gravitee" level="DEBUG" />
```

Comment:
Is it an intentional log level change from INFO to DEBUG?

Reply:
no